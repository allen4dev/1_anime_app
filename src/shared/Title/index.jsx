import React from 'react';
import './index.css';

/* eslint-disable */
function Title() {
  return <h1 className="Title">Simple Test</h1>;
}

export default Title;
