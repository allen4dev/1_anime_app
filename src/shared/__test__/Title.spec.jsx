import React from 'react';
import { shallow } from 'enzyme';

import Title from './../Title';

test('Title should have the text passed', () => {
  const title = shallow(<Title />);
  expect(title).toMatchSnapshot();
});
