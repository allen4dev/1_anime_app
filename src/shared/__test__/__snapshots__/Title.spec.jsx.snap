// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should have the text passed 1`] = `
<h1
  className="Title"
>
  Simple Test
</h1>
`;
